From d11c56d0feb1c79eefbc84e0050d8c10ef5bf45a Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Sat, 25 Jul 2026 15:52:59 -0400
Subject: [PATCH] version bump EXTENDNO to Reaper_v1.7.9

v1.7.9 = e7838c869c (2.4GHz hex fix for MLO-excluded band) + cc0117e1d6 (VPN Fusion device-glyph CSS scope fix) + d5896f6850 (speed test silent auto-retry).
---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index a1587d13cec..9b18ff18d83 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v1.7.8
+EXTENDNO=Reaper_v1.7.9
 
 RCNO=0
-- 
2.25.1

